client/src/components/Button.js

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+
+const Button = styled.button`
+  text-align: center;
+  width: 280px;
+  height: 66px;
```

Comment:
please be careful with fixed `width` and `height` this will greatly limit responsiveness and likely become an issue later on. Let me know if you want to chat about this